test/functional/tests/performance/test_100p_hits.py

```diff
@@ -49,24 +53,47 @@ def test_4k_100p_hit_reads_wt(queue_depth, numjobs, cache_line_size, perf_collec
         .io_depth(queue_depth)
         .cpus_allowed(get_dut_cpu_physical_cores())
         .direct()
+        .io_size(size_per_job)
     )
+    # spread jobs
+    offset = (testing_range / numjobs).align_down(Unit.Blocks512.value)
```

Comment:
Is the testing_range correct for calculating the offset? testing_workset value was used here before.

Reply:
testing_workset is larger than the core, so part of the offsets was actually outside of core bounds, and the jobs were ignored by fio